test/images/pets/peer-finder/README.md

```diff
@@ -0,0 +1,39 @@
+# Peer finder
```

Comment:
is this image only used for e2e tests? or is it also used more generally?

if it's only for tests, it seems reasonable to move here, but if it has uses outside testing I'm not sure that `test/images` is the right place for it.